### ISSUE TITLE
[Desktop][STG][Front End][bug] Informational text under “Medical Help”, and “Other Help” label has low color contrast. 

### ISSUE THREAD

**Issue by ghost** [FightPandemics/FightPandemics#2135]
Environment:
macOS 10.14 Mojave
MacBook Pro (15-inch, Early 2015)
Browser: Firefox 78.6.1esr 
https://staging.fightpandemics.work/

Steps:
1. Navigate to https://staging.fightpandemics.work/
2. Login to FP
3. Click on profile initial icon on top left corner of page. and click “view your profile” option
4. Click on the pencil (edit) located on the left corner 
5. On the next page click on “Edit Account” option located on right side of page
6. Use [TAB] key to to navigate Edit Account page
7. Observe low color contrast on text-leaf  label (div) below the “Medical Help”, and “Other Help”

Expected Behavior: 
Informational text should be visible to every type of user (accessibility violation - color contrast ratio should be greater then 4.5<. (accessibility violation)

Actual Behavior:
 Informational text under “Medical Help”, and “Other Help” (div) label has low color contrast.  Actual ratio 3.07 Expected ratio 4.501< .

![color-contrast FP](https://user-images.githubusercontent.com/76545271/104430236-44c31880-553b-11eb-9bed-b8741351291f.png)
